db/migrate/20231215181620_add_guests_to_rooms.rb

```diff
@@ -0,0 +1,5 @@
+class AddGuestsToRooms < ActiveRecord::Migration[7.0]
+  def change
```

Comment:
Same idea here -- a Room can exist without any guest information so we don't want a room_id foreign key on this table. The only foreign keys for this relationship should live on the stays table